drivers/CoreSPI/include/CoreSPI.h

```diff
@@ -0,0 +1,33 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
```

Comment:
pour @YannLocatelli  ;)

```suggestion
// Copyright 2022 APF France handicap
```